tools/loop-back-lvm/BUILD.bazel

```diff
@@ -0,0 +1,63 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
```

Comment:
is `tools` the right place for this?  I think of `tools` as a place for utility programs (executables).  Would it be better to have this in a directory alongside the builder docker image?

Reply:
I looked at this as a utility container to enable ember to work. But I have no strong feelings on where to put it.